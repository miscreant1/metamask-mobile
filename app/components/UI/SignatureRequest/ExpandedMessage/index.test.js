import React from 'react';
import { shallow } from 'enzyme';
import ExpandedMessage from './';

describe('ExpandedMessage', () => {
	it('should render correctly', () => {
		const wrapper = shallow(<ExpandedMessage currentPageInformation={{ title: 'title', url: 'url' }} />);
		expect(wrapper.dive()).toMatchSnapshot();
	});
});
