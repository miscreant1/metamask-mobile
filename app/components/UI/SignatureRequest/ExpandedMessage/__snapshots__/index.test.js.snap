// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpandedMessage should render correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "borderTopLeftRadius": 20,
      "borderTopRightRadius": 20,
      "flex": 1,
      "maxHeight": "75%",
      "paddingBottom": 20,
      "paddingHorizontal": 16,
      "paddingTop": 16,
    }
  }
>
  <TouchableOpacity
    activeOpacity={0.2}
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "space-between",
        "marginBottom": 20,
        "width": "100%",
      }
    }
  >
    <Icon
      allowFontScaling={false}
      name="ios-arrow-back"
      size={30}
      style={
        Object {
          "color": "#bbc0c5",
          "flex": 1,
        }
      }
    />
    <Text
      style={
        Object {
          "flex": 1,
          "fontFamily": "CircularStd-Medium",
          "fontSize": 16,
          "fontWeight": "600",
          "textAlign": "center",
        }
      }
    >
      Message
    </Text>
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    />
  </TouchableOpacity>
  <View
    style={
      Object {
        "alignItems": "center",
        "marginBottom": 20,
      }
    }
  >
    <WebsiteIcon
      style={
        Object {
          "borderRadius": 20,
          "height": 40,
          "marginBottom": 20,
          "width": 40,
        }
      }
      title="url"
      url="url"
    />
    <Text
      style={
        Object {
          "fontFamily": "CircularStd-Medium",
          "fontSize": 16,
          "fontWeight": "600",
          "textAlign": "center",
        }
      }
    >
      Message from
       
      url
    </Text>
  </View>
  <ScrollViewMock
    style={
      Object {
        "flex": 1,
      }
    }
  />
</View>
`;
